src/helpers/linkHelper.js

```diff
@@ -20,7 +20,12 @@ function ensureProtocol(link) {
     linkWithProtocol = `http://${link}`;
   }
 
-  return linkWithProtocol;
+  return spaceDeletion(linkWithProtocol);
```

Comment:
i suggest `link.replaceAll(' ', '');` when setting `let linkWithProtocol = link` instead of creating a method that is used only once? and i see no reason in removing spaces from `linkWithProtocol` as it is the `link` prefixed with http://